var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1201,3 +751,435 @@ def test(self):
         self.run_test(
             exe, options=options, expected=[self.spec.prefix], installed=True, purpose=reason
         )
+
+
+class BuildEnvironment(object):
```

Comment:
Nitpick (we are not supporting Python 2.7 anymore):
```suggestion
class BuildEnvironment:
```

Reply:
Right! I ported this prior to the py2 drop and haven't revisited in a few months. Great!